src/IW/App/Error.hs

```diff
@@ -120,6 +144,11 @@ toHttpError (AppError _callStack errorType) = case errorType of
         MissingHeader name     -> err401 { errBody = toLazy $ "Header not found: " <> foldedCase name }
         HeaderDecodeError name -> err401 { errBody = encodeUtf8 $ "Unable to decode header: " <> name }
         DbError e              -> err500 { errBody = encodeUtf8 e }
+    GitHubError err -> case err of
```

Comment:
I think here instead of 4 cases you can just do `show` and return `err500`. Should be okay for now.